_includes/nav.html

```diff
@@ -20,7 +20,11 @@
 
             <li class="nav__link"> <a href="http://us11.campaign-archive1.com/home/?u=e0d8362a03edb98f9391ddcd5&id=72d2723bcf">News</a></li>
             <li class="nav__link"> <a href="https://ardor.zendesk.com">Help</a></li>
-            <li class="nav__link"> <a href="https://accounts.ardoreducation.com">Login</a></li>
+            <li class="nav__link">
+                <a href="https://accounts.ardoreducation.com/#/login">Sign in</a>
+                &nbsp; or &nbsp;
```

Comment:
I the extra space doesn't do anything. I just thought it was hard to read. I can remove them.